### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by ragangithub**
- Added CSS to style the website .

### CODE REVIEW

File: src/components/BookItem.js [hunk 1]
```diff
@@ -2,24 +2,54 @@ import React from 'react';
...
 const BookItem = ({ itemProp }) => {
   const dispatch = useDispatch();
 
   return (
     <>
-      <li>{itemProp.title}</li>
-      <li>{itemProp.author}</li>
+      <li className="book-cards">
+        <div className="book-info">
+          <p className="category">{itemProp.category}</p>
```
Comment (souad988) [inline src/components/BookItem.js:14]: Greate work done so far, I would just mention that the category is missing in you cards , you can add a static values to make your design much with the template , also please reduce the vertical space between the titles and buttons refer to the images below 

image (19)
image (24)

File: src/components/InputBook.js [hunk 2]
```diff
@@ -36,28 +37,31 @@ const InputBook = () => {
...
-          value={data.author}
-          onChange={handleChange}
-        />
-        <button type="button" className="addBook" onClick={handleSubmit}>
-          Add Book
-        </button>
-      </form>
+      <hr className="form-hr" />
+      <div className="add-book">
+        <h3>ADD NEW BOOK</h3>
```
Comment (souad988) [inline src/components/InputBook.js:42]: Kindly use the same fonts and colors used in the template the title and divider looks diffrent here, also increase left marging of your component 

image (18)
image (23)

File: src/components/Navbar.js [hunk 3]
```diff
@@ -1,20 +1,27 @@
...
+        <ul className={styles.navlinks}>
+          <li className={styles.links}>
+            <Link to="/">Books</Link>
+          </li>
+          <li className={styles.linksCat}>
+            <Link to="/categories">Categories</Link>
+          </li>
+        </ul>
+      </div>
+      <div className={styles.profile}>
```
Comment (souad988) [inline src/components/Navbar.js:20]: Well done by adding the profile icon but please do your best to make it look like the design in terms of size and color 

image (21)

image (26)

File: src/components/InputBook.js [hunk 4]
```diff
@@ -36,28 +37,31 @@ const InputBook = () => {
...
+        <form action="submit" className="submit">
```
Comment (souad988) [inline src/components/InputBook.js:43]: Please this form should take the full width of the ùain section just like it's upper component and inputs should fill the space 

image (16)
image (17)

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @ragangithub  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


242722065-db428b0c-8dfc-49c4-a156-42f227e32868


## Highlights
Good file structure
No inline styling
Good class names

## Required Changes ♻️
_Check the comments under the review._
_ Also please change the background color as it looks darker than the design , and increase margin of your main section to much with the template 
image (16)
image (17)

 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ragangithub** [comment]
- Changed the background color  and increased the margin of the main section to much with the template
- Included category and also reduced  the vertical space between the titles and buttons 
- Used the same fonts and colors used in the template and corrected the divider 
- Increased the size of the profile icon and used the same color as in the design
- Form takes the full width

**Review by binhussen — APPROVED** [review]
Hi @ragangithub , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Beautiful UI ✔️
- Well Structured Code ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**